apps/client/src/common/components/multi-part-progress-bar/MultiPartProgressBar.scss

```diff
@@ -0,0 +1,27 @@
+@use '../../../theme/viewerDefs' as *;
+
+$progress-bar-size: 12px;
+$progress-bar-br: 6px;
```

Comment:
the height of the colored bars is inherited by the parent div .... so it cant be bigger. 2px to what? border radius?

Reply:
I mean in height. It could be a visual trick from the colour difference. But if you zoom in or take a screenshot, the white bit is taller than the red

Reply:
can't say that i could reproduce this 😆